plugin.xml

```diff
@@ -92,8 +92,20 @@ xmlns:android="http://schemas.android.com/apk/res/android"
         <framework src="com.google.firebase:firebase-analytics:$ANDROID_FIREBASE_ANALYTICS_VERSION" />
         <framework src="build.gradle" custom="true" type="gradleReference" />
 
-        <source-file src="src/android/FirebaseAnalyticsPlugin.java"
-            target-dir="src/by/chemerisuk/cordova/firebase/" />
+        <source-file src="src/android/FirebaseAnalyticsPlugin.java" target-dir="src/by/chemerisuk/cordova/firebase/" />
```

Comment:
Since we're changing this, maybe we could rename the target directory (it looks like something from the repo we forked from).